app/driver.go

```diff
@@ -371,6 +387,33 @@ func discoverFlexvolumeDir(kubeClient *clientset.Clientset) (dir string, err err
 	return value, nil
 }
 
+func startProvisioner(kubeClient *clientset.Clientset, managerURL string) error {
+	logrus.Debug("Enable the built-in Longhorn provisioner only for FlexVolume")
+
+	clientOpts := &longhornclient.ClientOpts{Url: managerURL}
+	apiClient, err := longhornclient.NewRancherClient(clientOpts)
+	if err != nil {
+		return errors.Wrap(err, "Cannot start Provisioner: failed to initialize Longhorn API client")
+	}
+
+	serverVersion, err := kubeClient.Discovery().ServerVersion()
+	if err != nil {
+		return errors.Wrap(err, "Cannot start Provisioner: failed to get Kubernetes server version")
+	}
+	provisioner := controller.NewProvisioner(apiClient)
+	pc := pvController.NewProvisionController(
+		kubeClient,
+		controller.LonghornProvisionerName,
+		provisioner,
+		serverVersion.GitVersion,
+	)
+
+	//FIXME stopch should be exposed
```

Comment:
I've forgotten this comment is there. Now we should able to use `util.RegisterShutdownChannel(done)` like what we do with Flexvolume and CSI.